config/tests.conf.php

```diff
@@ -65,6 +65,9 @@
                     'type' => 'keyword',
                     'ignore_above' => 256,
                 ],
+                'referenced_resources' => [
```

Comment:
Lets not forget to add the test identifier

Reply:
Done 63d3128532d69e59a7626583c8417aa6410ef439